src/components/StartButton.js

```diff
@@ -1,12 +1,23 @@
-import newQuote from '../utils/quoteGenerator';
 import '../css/App.css';
+import newQuote from '../utils/quoteGenerator';
 
 const StartButton = (props) => {
+
+    const handleClick = (event) => {
+        console.log(event);
```

Comment:
If you're happy with the event it's good practice to not let `console.logs` get through the PR stage, and you'll often find that a lot when working in bigger dev teams!